app/Http/Resources/V2/Dashboard/ViewRestorationStrategyResource.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Resources\V2\Dashboard;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class ViewRestorationStrategyResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'restorationStrategies' => $this->resource['restorationStrategies'] ?? null,
+            'landUseTypes' => $this->resource['landUseTypes'] ?? null,
+            'landTenures' => $this->resource['landTenures'] ?? null,
```

Comment:
Likewise, this class isn't doing anything super useful (and same for the one below). This structure can just get returned directly from the controller.

Reply:
removed unnecesary parsing